src/api/procedures/createAsset.ts

```diff
@@ -224,6 +230,18 @@ export async function prepareCreateAsset(
     );
   }
 
+  if (initialStatistics?.length) {
```

Comment:
Even though currently there are no protocol fees for setting active stats, there might be in the future. You should use the `addManualFees` "trick" we're using for all other transactions in this procedure (remember to change the comment to reflect this case)

Reply:
Looking back, calling `addManualFees` once per every transaction we add is cumbersome. I think it makes more sense to store all the tags in an array and just make a single call before the `addBatchTransaction` call

Reply:
We are storing the transactions in an array, so with a lookup table, we can map from transactions to tags.

Reply:
After trying it out, I think we should keep a separate array like you suggested, 

Mapping from `SubmittableExtrinsic` types is vague and harder to reason about. Using `toString` could work, but the extra variable is cleaner imo.